src/main.tsx

```diff
@@ -1,6 +1,11 @@
 import ReactDOM from "react-dom/client";
 import App from "./app";
+import { GoogleOAuthProvider } from "@react-oauth/google";
+
+import {CLIENT_ID} from "./app/api/config"
 
 ReactDOM.createRoot(document.getElementById("root") as HTMLElement).render(
-  <App />,
+  <GoogleOAuthProvider clientId={CLIENT_ID!}>
```

Comment:
We have folder providers and we need to write this logic in a separate file in that folder

Reply:
done